ramls/metadata-provider.raml

```diff
@@ -242,4 +243,17 @@ resourceTypes:
           body:
             text/plain:
               example: "Internal server error"
+  /incomingRecords/{recordId}:
+    get:
+      description: get incoming record DTO by id
+      responses:
+        200:
+          body:
+            application/json:
+              type: incomingRecordDto
```

Comment:
propose to return incomingRecord itself instead of returning it as String value for this dto

Reply:
good point. discussed offline, adhering to this approach